src/components/Recaptcha.jsx

```diff
@@ -5,11 +5,11 @@ function Recaptcha({ onToken }) {
 
   useEffect(() => {
     const script = document.createElement("script");
-    script.src = "https://www.google.com/recaptcha/api.js?render=6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE";
+    script.src = "https://www.google.com/recaptcha/api.js?render=6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP";
```

Comment:
Remove the code, add it to the .env file and call the env variable here to use it